### DISCUSSION TITLE
Logo design

### DISCUSSION THREAD

**Discussion by leikoilja** [leikoilja/ha-google-home#49] [Archive]
Hey guys, @ArnyminerZ, @DurgNomis-drol 💥 

Does any of you have the passion and desire to help with a logo for the integration? 😄 
To publish to HACS we need a logo, here is the [reference](https://hacs.xyz/docs/publish/integration#home-assistant-brands). We need a logo and it needs to follow all requirements that HACS requires. Plus PR open and merged for `brands` HACS repo. 

It can be anything from a simple google home logo to anything custom 😄  Does anyone has a desire to pick it up?

**Comment by DurgNomis-drol** [comment]
I will take a crack at it later today or tomorrow 😁

**Reply by leikoilja** [reply]
Yay! Awesome! If you need any input or second opinion feel free to drop it here first ;)

**Reply by DurgNomis-drol** [reply]
Would it be totally out of place to just use the logo that Google Home uses? As this integration actually uses the same API as Google home does to control the devices? 

Or should i add some like `Local API` or just `Local`?

![icon](https://user-images.githubusercontent.com/67142049/109990054-58078e80-7d09-11eb-895f-098847c87e4b.png)

**Reply by leikoilja** [reply]
I think it's okay if we use google home logo 😄  copy cats.
I dont really know about "local API" or "local". I don't mind, but would it add any value? Or should we just use google logo and make it fit all the requirements HA asks for and settle on that? 😄

**Reply by DurgNomis-drol** [reply]
I don't think adding "local" or "local API" would add anything to the logo, and in future we might be implementing other things from Google Home, so i think we should make the Official logo fit and be done with it. 😄 

@ArnyminerZ What do you think?

**Reply by leikoilja** [reply]
Sounds goood to me :)

**Reply by ArnyminerZ** [reply]
I'd simply use the Google Home logo. It would be more intuitive for the people. And the name can also be simply "Google Home" or "Home Assistant Google Home"/"HA Google Home"

**Comment by DurgNomis-drol** [comment]
I have opened a pull request at [Homeassistant/Brands](https://github.com/home-assistant/brands/pull/2317)

**Reply by leikoilja** [reply]
Awesome! Thanks 🤩💥 I see it got merged, that's amazing

**Reply by DurgNomis-drol** [reply]
Thanks, I think we should see our new logo working in a hour or so in our local HA instance 😄

**Reply by ArnyminerZ** [reply]
Awesome! Great work!
